build.gradle

```diff
@@ -43,7 +44,7 @@ repositories {
 }
 
 dependencies {
-    compile("org.spongepowered:mixin:0.7.11-SNAPSHOT")
+    implementation("org.spongepowered:mixin:0.7.11-SNAPSHOT")
```

Comment:
file size could be reduce by excluding modules like guava gson, launchwrapper, commons-io